SI/Task09.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+
+int Broqch(char* arr)
+{
+	int br = 0;
+	while (arr[br] != '\0')
+	{
+		br++;
+	}
+	return br;
+}
+
+int Index(char* text, char* word)
+{
+	int pom = -1;
+	for (int i = 0; i < Broqch(text); i++)
```

Comment:
Big mistake - fix the for loop - you are calculating the length of the char[] after every iteration of the for loop
=> size_t length = myStrLen(text) and then use length in the condition of the for loop